tests/unit/test_tags_parser.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from pytest_splunk_addon.standard_lib.addon_parser.tags_parser import (
+    TagsParser,
+    unquote,
+)
+
+
+output_to_build = {
+    "eventtype=fiction_for_tags_positive": {
+        "tags_positive_event": "enabled",
+        "tags_disabled_event": "disabled",
+    },
+    "source=%2Fopt%2Fsplunk%2Fvar%2Flog%2Fsplunk%2Fsplunkd.log": {
+        "tags_positive_event": "enabled",
+        "tags_disabled_event": "disabled",
+    },
+}
+
+
+def test_tags_can_be_parsed_and_extracted(parser_instance):
+    assert hasattr(
+        parser_instance.tags, "sects"
```

Comment:
It looks more like precondition to test. It is possible that `parser_instance.tags == []` for example